### PR TITLE
[LOOP-4882] Mute App Sounds Enhancements

### PR DESCRIPTION
**Pull request by Camji55**
https://tidepool.atlassian.net/browse/LOOP-4882

### CODE REVIEW

File: Loop/Views/IOSFocusModesView.swift [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                ForEach(Array(zip(bullets.indices, bullets)), id: \.0) { index, bullet in
+                    HStack(spacing: 10) {
+                        NumberCircle(index + 1)
+                        
+                        Text(bullet)
+                    }
+                }
+                
+                // MARK: To be removed before next DIY Sync
+                if appName.contains("Tidepool") {
```
Comment (Camji55) [inline Loop/Views/IOSFocusModesView.swift:48]: @ps2 lmk how you feel about this solution

Comment (ps2) [reply Loop/Views/IOSFocusModesView.swift:48]: Can we just put the images in the assets *override* area in Tidepool's LoopWorkspace?  I think that would be preferrable.

Comment (Camji55) [reply Loop/Views/IOSFocusModesView.swift:48]: Okay, I've moved them!

**Review by ps2 — APPROVED** [review]
LGTM!

**Pull request merged**